HHS/OPRE-OPS
# OPS-1449 Refactor agreement/{id}/history endpoint

## What changed

This is one small part of the tech debt item OPS-1449 where we are trying to restrict the depth our rest API goes to when nesting routes. This PR fixes one route, the agreements/{id}/history endpoint to be agreement-history/{id} so that the endpoint matches our standard defined in the tech debt.

## Issue

[OPS-1449](https://github.com/HHS/OPRE-OPS/issues/1449)

## How to test
1. Run Python unit tests
2. Log in as basic user, then open any agreement. Request to move a budget line to planned or in execution.
3. Log in as division director, then approve that agreement.
4. Open the agreement and look at the agreement history. You should still see at least the 2 items corresponding to this interaction in the agreement history page.

## Screenshots

![image](https://github.com/user-attachments/assets/9f15d545-96d8-4137-a53e-d02bd93c8646)
A screenshot of the agreement history  on localhost after my refactor. If you are testing this change, you should also see something similar. 

## Definition of Done Checklist
